src/App.tsx

```diff
@@ -15,48 +17,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortString {
+  Alphabet = 'alphabet',
+  Length = 'length',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: string,
+  isReverse: boolean,
+): string[] {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortString.Alphabet:
+          return good1.localeCompare(good2);
+
+        case SortString.Length:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReverse) {
+    return preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setsortField] = useState('');
```

Comment:
need provide default value

Reply:
empty string is an default value
Why not?